docker-compose.yml

```diff
@@ -1,23 +1,27 @@
 version: '3'
 services:
     mysql:
-        image: mysql
-
+        image: mysql/mysql-server
         environment:
-            - MYSQL_ROOT_PASSWORD=
+            - MYSQL_ROOT_PASSWORD
+            - MYSQL_DATABASE=lidarrupdate
+            - MYSQL_USER=root
+            - MYSQL_PASSWORD=${MYSQL_ROOT_PASSWORD}
+        ports:
+            ## Specify a port here so that you can connect to the mysql server from the host
+            - 33060:3306
 
```

Comment:
Note: This port here is only if you want connect into your mysql from your HOST (windows / linux ) without `exec`ing into the container.. can be removed if you don't think it's necessary

Reply:
I think it may be best to leave it out for the moment unless we need outside access to the container. We could always put it back in if we come across a need to access the db from outside.